tests/pipeline/Jenkinsfile.k8sSetup

```diff
@@ -0,0 +1,116 @@
+                    env.versionId = readFile('version').trim()
+                    sh'''
+                          #!/bin/bash
+                          docker load -i rook.tar.gz
+                          docker load -i toolbox.tar.gz
+                          docker images
+                          rookImg=$(docker images |grep rook | awk '{print  $1}')
+                          echo $rookImg
+                          toolboxImg=$(docker images |grep toolbox | awk '{print  $1}')
+                          echo $toolboxImg
+                          echo "helm Image:"
+                          echo ${versionId}
+                          docker tag $rookImg rook/rook:master
+                          docker tag $rookImg rook/rook:${versionId}
+                          docker tag $toolboxImg rook/toolbox:master
+                          '''
+
+               }
+
+               stash name: 'test-scripts',includes: 'tests/scripts/'
+
+            }
+        }
+        //start 3 node k8s cluster
+        stage ("Set Up k8s") {
+            steps{
+                script{
+                    try{
+                        withEnv(["KUBE_VERSION=${env.kube_version}"]){
+                            echo "setting up 3 node k8s ${env.kube_version} cluster"
+                            sh '''#!/bin/bash
+                                  echo " setting up master"
+                                  export KUBE_VERSION='''+ "${env.kube_version}" + '''
+                                  kubeadm_join_flags="$(tests/scripts/kubeadm.sh install master | grep '^ *kubeadm join' | sed 's/^ *kubeadm join //')"
+                                  echo ${kubeadm_join_flags} > ktoken'''
+                            script{
+                                token = readFile('ktoken').trim()
+                            }
+                            node("${params.node}-n1"){
+                                unstash 'test-scripts'
+                                sh '''#!/bin/bash
+                                      echo " setting up node 1"
+                                      export KUBE_VERSION='''+ "${env.kube_version}" + '''
+                                      tests/scripts/kubeadm.sh install node ''' + "${token}"
+                            }
+                            node("${params.node}-n2"){
+                                unstash 'test-scripts'
+                                sh '''#!/bin/bash
+                                      echo " setting up node 2"
+                                      export KUBE_VERSION='''+ "${env.kube_version}" + '''
+                                      tests/scripts/kubeadm.sh install node ''' + "${token}"
+                            }
+                            sh '''#!/bin/bash
+                                  export KUBECONFIG=$HOME/admin.conf
+                                  tests/scripts/kubeadm.sh wait 3'''
+                        }
+                    }finally{
```

Comment:
You have this finally clause that is really not doing anything, remove?

Reply:
missed cleaning it up after refactor. Should be good now.